docs/api/reservation.md

```
@@ -0,0 +1,94 @@
+
+## 예약 조회 API
+
+### Request
+
+```
+GET /reservations?themeId=1&memberId=1&dateFrom='2023-08-05'&dateTo='2023-08-05' HTTP/1.1
+```
+
+### Response
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)